tests/testthat/test-queue.R

```diff
@@ -53,7 +53,7 @@ test_that("queue works as intended", {
   expect_length(queue$queue$task_list(), 0)
 
   con <- queue$queue$con
-  key <- queue$queue$keys$worker_name
+  key <- r6_private(queue$queue)$keys$worker_name
```

Comment:
This feels like we should be using something in rrq directly to test how many workers are on a queue; worth speccing up?

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4062/Add-a-function-to-get-number-of-workers-listening-on-a-queue

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4062/Use-rrq-API-function-to-get-workers-from-queue updated with api function now

Reply:
Actually I think we need this as we're testing here that workers get cleaned up when the `queue` object is removed. So we need a handle on the worker name key which will work after the `queue` object has been removed. https://github.com/mrc-ide/hintr/pull/458/files#diff-55e2df69a210ae4956f1697039076156121e0f0684eaf9441271f031b69f204bL62 this line